voorhoede/head-start
# chore: generate & use i18n types

# Changes

Adds autocomplete to `t()` function based on available translations in DatoCMS. For example `t('play_video')`. 
These suggestions are provided by generating a `TranslationKey` enum which is based on the `key`s of translation records in DatoCMS.

# Associated issue

N/A

# How to test

1. Checkout this branch.
2. Use the `t()` function.
3. Verify you're getting suggestions based on the `TranslationKey`s from the generated `i18n.types.ts` file.

# Checklist

- ~~I have added/updated tests to prove that my feature works (if not possible please explain why)~~
- ~~I have made changes to the README and if the change affects the project setup (npm commands changed, new service added, environmental variable added)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
